### PR TITLE
Integration tests added

### PR DESCRIPTION
**Pull request by ilivinskyi**

### CODE REVIEW

File: src/test/java/com/ilivinskyi/glovo/glovo/OrderControllerTests.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import static com.ilivinskyi.glovo.glovo.TestData.DefaultOrderData.getOrderJsonString;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@TestPropertySource(
+        locations = "classpath:application-local.properties")
+class OrderControllerTests {
```
Comment (mkharkhuta) [inline src/test/java/com/ilivinskyi/glovo/glovo/OrderControllerTests.java:23]:
```suggestion
class OrderControllerTest {
```

File: src/test/java/com/ilivinskyi/glovo/glovo/OrderControllerTests.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+    @Autowired
+    private OrderRepository orderRepository;
+
+    @Test
+    void testGetOrder() throws Exception {
+        var order = getDefaultOrder();
+        orderRepository.save(order);
+        mockMvc.perform(get("/orders/" + order.getId()))
+                .andExpect(status().isOk());
```
Comment (mkharkhuta) [inline src/test/java/com/ilivinskyi/glovo/glovo/OrderControllerTests.java:36]: Other fields validation would be nice too

File: src/test/java/com/ilivinskyi/glovo/glovo/OrderControllerTests.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    @Test
+    void testDeleteOrder() {
+        var order = getDefaultOrder();
+        var savedOrder = orderRepository.save(order);
+        var orderId = savedOrder.getId();
+        mockMvc.perform(delete("/orders/" + orderId)
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isNoContent());
+    }
+}
```
Comment (mkharkhuta) [inline src/test/java/com/ilivinskyi/glovo/glovo/OrderControllerTests.java:110]: Would be nice to add negative and aime edge cases too 🙂

**Review by mkharkhuta — APPROVED** [review]
Good 👍
